sound.js

```diff
@@ -170,13 +175,10 @@ Sound.prototype.setSpeed = function(value) {
   if (this._loaded) {
     if (!IsWindows && !IsAndroid) {
         RNSound.setSpeed(this._key, value);
-    }else if (IsAndroid) {
-      const isPlayingPromise = RNSound.isPlaying(this._key);
-      isPlayingPromise.then((result) => {
```

Comment:
The other develop cannot understand `result`. `successful` or `isSuccessful` is the better choise.